packages/mobile/src/navigator/types.tsx

```diff
@@ -9,7 +9,14 @@ import { TransactionDataInput } from 'src/send/SendAmount'
 import { ReviewProps } from 'src/transactions/TransactionReview'
 import { TransferConfirmationCardProps } from 'src/transactions/TransferConfirmationCard'
 
-// tslint:disable-next-line
+// Typed nested navigator params
+type NestedNavigatorParams<ParamList> = {
```

Comment:
bravo

Reply:
Thanks! Had to think carefully about this 🤯 😄